tck/src/main/java/org/eclipse/microprofile/openapi/tck/ModelConstructionTest.java

```diff
@@ -311,12 +311,16 @@ public void pathsTest() {
         final String pathItemKey = "myPathItem";
```

Comment:
Not directly related to this change.

Current javadoc of `addPathItem(..)` says:
> name a path name in the format valid for a Paths object. The field name MUST begin with a slash.

But the TCK adds a `PathItem` without leading slash.

Should `pathItemKey` be changed to add `"/myPathItem"` instead?

---

Should an other test be created that ensure that `"somePath"` can not be added? In this case, what is the type of exception that should be expected by the TCK test?

---
Current Javadoc (not changed with this PR) for reference:
https://github.com/eclipse/microprofile-open-api/blob/8e0f2479a00470911d64ae13a59c5cd78f4240c9/api/src/main/java/org/eclipse/microprofile/openapi/models/Paths.java#L34-L41

Reply:
I have changed my mind on this matter:

The models can contain malformed OpenAPI instances. 

If at TCK level we enforce the OpenAPI Spec to be valid, this means that the models classes can no hold malformed Spec. This is used by Parsers to report "Errors" and "Warnings".

Reply:
good point @jmini.   My suggestion is to make the TCK test a valid OAS instance (so using `/myPathItem`).   We haven't covered in the MP spec how to deal with errors and warnings so at the moment it's implementation dependent.  This is probably a great topic for discussion though, and potential `2.0` item.